commons/src/graph.rs

```diff
@@ -32,25 +32,34 @@ impl Graph {
     pub fn from_metadata(
         releases: Vec<metadata::Release>,
```

Comment:
Side note, maybe we can use https://github.com/coreos/stream-metadata-rust in the future.